From: Reinhard Tartler <siretart@tauware.de>
Date: Fri, 30 Aug 2024 21:40:49 -0400
Subject: relax env_logger dependency

---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 9b6601b..6d4601e 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -32,7 +32,7 @@ deps-serde = ["chrono/serde", "url/serde"]
 [dependencies]
 anyhow = "1.0.86"
 clap = { version = ">= 4.4", features = ["derive", "env"] }
-env_logger = "0.11.5"
+env_logger = { version = ">= 0.10" }
 ipnet = { version = "2.9.0", features = ["serde"] }
 iptables = "0.5.2"
 libc = "0.2.154"
