From: Reinhard Tartler <siretart@tauware.de>
Date: Fri, 30 Aug 2024 21:42:36 -0400
Subject: Relax nispor dependency

---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 22b1299..f014e69 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -60,7 +60,7 @@ prost = "0.12.6"
 futures-channel = "0.3.30"
 futures-core = "0.3.30"
 futures-util = "0.3.30"
-nispor = "1.2.19"
+nispor = { version = ">= 1.2" }
 tower = { version = "0.4.13" }
 
 [build-dependencies]
